compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -104,5 +124,82 @@ CircleConst *create_circleconst(GraphBuilderContext *context, int32_t tensor_ind
 
   return const_node;
 }
+#endif // 0
+
+//
+// circleconst_from_tensor() ?
+//
+CircleConst *create_circleconst(GraphBuilderContext *context, int32_t tensor_index)
+{
+  LOGGER(l);
+
+  auto graph = context->graph();
+  auto reader = context->reader();
+  const auto &tensors = reader->tensors();
+  const circle::TensorT &const_tensor = *tensors[tensor_index];
+
+  const std::vector<uint8_t> &buffer = reader->buffers()[const_tensor.buffer]->data;
+  std::vector<int32_t> const_dims = const_tensor.shape; // in NHWC
+  if (const_dims.size() == 0 && buffer.empty())
+  {
+    // unknown shape tensor
+    return nullptr;
+  }
+
+  uint32_t num_elements = 1;
+  for (uint32_t r = 0; r < const_dims.size(); ++r)
+  {
+    num_elements = num_elements * const_dims[r];
+  }
+
+  if (buffer.empty() && num_elements > 0)
+  {
+    // normal empty tensor
```

Comment:
I don't know this.. but above `if (const_dims.size() == 0 && buffer.empty())` conditions is normal empty tensor I think. But, this condition(buffer is empty but num elements has no zero) looks weird. Could you let me know about this network or case?

And, (1, 8, 0, 64) tensor won't evaluate true to this condition because it is (buffer emtty and num_elements is zero). Am I right?

Reply:
I'm not sure what your question is. I'll ask you in off-line.